ci/Jenkinsfile.linux

```diff
@@ -44,34 +45,34 @@ pipeline {
   stages {
     stage('Prep') {
       steps {
-        script {
+        script { nvm(env.NODE_VERSION) {
```

Comment:
> We might have to update the Dockerfile to install NVM for Jenkins.

I think that is the right thing to do because we might want to be able to upgrade node versions separately, say, for release branches and for develop. So NVM is pretty much a requirement for that.

Reply:
@jakubgs, was asking same about nvm on Linux in Status private chat but missed this conversation. Yes, Linux build fails without it.